addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_sampler.py

```diff
@@ -103,6 +103,9 @@ def __gather_wrap(blender_shader_node, export_settings):
         wrap_s = TextureWrap.Repeat
     wrap_t = wrap_s
 
+    # Starting Blender 3.5, MIRROR is now an extension of image node
+    # So this manual uv wrapping trick is no more usefull
+    # Keeping it for now, for compatibility
```

Comment:
This isn't correct. The manual wrapping is still needed when the wrapS and wrapT are not the same.